CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# Changelog
+
+
+## 1.0.1 -- 2019-09-27
+* Modified Doxygen config file to generate dependency graphs [#376](https://github.com/aws/aws-encryption-sdk-c/pull/376)
+* Ran reformat [#426](https://github.com/aws/aws-encryption-sdk-c/pull/426)
+* Strengthen memory_order for refcount_down [#433](https://github.com/aws/aws-encryption-sdk-c/pull/433)
+* Update PR template [#391](https://github.com/aws/aws-encryption-sdk-c/pull/391)
+* Refreshing the clang-format file and checking the version [#430](https://github.com/aws/aws-encryption-sdk-c/pull/430)
+* Fix MultiKeyringNew proof so it runs again. [#444](https://github.com/aws/aws-encryption-sdk-c/pull/444)
+* Windows build fixes [#446](https://github.com/aws/aws-encryption-sdk-c/pull/446)
+
```

Comment:
We should merge these two bug fixes as well along with this release change: 

- https://github.com/aws/aws-encryption-sdk-c/pull/408 
- https://github.com/aws/aws-encryption-sdk-c/pull/385

Is there any reason why we are not including these changes?

Reply:
They aren't approved?  My expectation was there would be several catch-up releases, but if these are ready, cool.